### ISSUE TITLE
manual install broken

### ISSUE THREAD

**Issue by nickcolossal** [barricklab/pLannotate#21]
error in manual install:
```
==> python setup.py install
['tabulate >=0.8.9', 'streamlit >=1.8.1', 'biopython>1.77', 'bokeh=2.4.1']
error in plannotate setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'=2.4.1'"
```

to fix changed:
https://github.com/barricklab/pLannotate/blob/03417a3991558fd2aef8cc68f9cf3d45853b0a6c/requirements.txt#L5

to be:
`bokeh==2.4.1`

**Comment by darachm** [comment]
And those should probably all be '==' , because streamlit put the cli within web, and in other news the script wants vegalite v4, and the latest is now 5, so that has to be explicit

etc

etc

So this works for an install line in docker:
`RUN cd pLannotate && sed -i 's/bokeh=2/bokeh==2/' requirements.txt && sed -i 's/>=/==/' requirements.txt && echo "altair==4.2.2" >> requirements.txt && python3 setup.py install  `

**Comment by jeffreybarrick** [comment]
Thanks!

I believe I've fixed this in daabf1a63be79c43ff6166c76403d358b1d53da8 by requiring exact versions of the dependencies, and moving all of this into the `environment.yml` file so conda will handle installing the Python modules versus `setup.py`.
